source/doc_versions.py

```diff
@@ -1,9 +1,9 @@
 # The API version strings to display in the documentation
 # COMMON_VERSION is the latest version of any one API, and is considered the overall API version
-COMMON_VERSION = '2016-06-12'
+COMMON_VERSION = '2016-07-01'
```

Comment:
Not if its an internal only use case. Once we make public facing changes (that affect the docs) we can publish the new docs. In the meantime, it will simply lead to confusion in customers. "You release a new version of your API, but the docs are the same? What?!?!?"